proof/records.go

```diff
@@ -43,8 +44,9 @@ const (
 	TapscriptProofTapPreimage2 tlv.Type = 3
 	TapscriptProofBip86        tlv.Type = 4
 
-	MetaRevealEncodingType tlv.Type = 0
-	MetaRevealDataType     tlv.Type = 2
+	MetaRevealEncodingType   tlv.Type = 0
+	MetaRevealDataType       tlv.Type = 2
+	MetaRevealDecimalDisplay tlv.Type = 5
```

Comment:
Should this actually be an even value? As if you don't understand it, then you won't be able to reconstruct the meta hash properly.

Reply:
No, even would mean you have to understand it or error out. With the "unknown odd types" functionality, you'll be able to construct the meta hash correctly, even if you don't understand each value. That was the whole point of allowing those unknown odd types. You can't decode their content if you haven't upgraded. But you can still calculate the hash of the TLV stream as a whole correctly.

Reply:
Yep, I was mistaken. I forgot about the addition where we'll just add the unknown TLV types to the buffer before we serialize.